docs2/site/docs/migrations/migration5.md

```diff
@@ -401,20 +401,22 @@ services.AddGraphQL(builder => builder
 
 ### 9. `ExecutionHelper.GetArgumentValues` was renamed to `GetArguments`
 
-### 10. `schema`, `variableDefinitions` and `variables` arguments were removed from `ValidationContext.GetVariableValues`
+### 10. `DirectiveGraphType` was renamed to `Directive`
```

Comment:
I recommend adding items to the end of the list.  We will need to cleanup the migration document anyway before releasing, and will logically group related changes at that time.  If you change the numbering in each PR, any other PR in progress that also changes numbering will have merge conflicts.  Ok with whatever though.